tests/Handler/CorsPreflightHandlerTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Tomaj\NetteApi\Test\Handler;
+
+use Nette\Http\Response;
+use Tomaj\NetteApi\EndpointIdentifier;
+use Tomaj\NetteApi\Handlers\CorsPreflightHandler;
+use Nette\Application\LinkGenerator;
+use Nette\Application\Routers\SimpleRouter;
+use Nette\Http\Url;
+use PHPUnit_Framework_TestCase;
+
+class CorsPreflightHandlerTest extends PHPUnit_Framework_TestCase
+{
+    public function testResponse()
+    {
+        $preflighthandler = new CorsPreflightHandler(new Response());
+        $result = $preflighthandler->handle([]);
+        $this->assertEquals(200, $result->getCode());
+        $this->assertEquals([], $result->getPayload());
+    }
+
+    public function testEndpointSetter()
+    {
+        $defaultHandler = new CorsPreflightHandler(new Response());
+        $this->assertNull($defaultHandler->getEndpoint());
+
+        $endpointIdentifier = new EndpointIdentifier('OPTIONS', 1, 'article', 'detail');
+        $defaultHandler->setEndpointIdentifier($endpointIdentifier);
+        $this->assertEquals($endpointIdentifier, $defaultHandler->getEndpoint());
+    }
+
+    /**
+     * @expectedException \Nette\InvalidStateException
+     */
+    public function testExceptionWhenCreatingLinkWithoutLinkGenerator()
+    {
+        $defaultHandler = new CorsPreflightHandler(new Response());
+        $defaultHandler->createLink([]);
+    }
+
+    /**
+     * @expectedException \Nette\InvalidStateException
```

Comment:
a had to remove this use statements, looks like there is problem with phpunit if i use use statement